centreon/src/Core/Dashboard/Application/UseCase/DeleteContactDashboardShare/DeleteContactDashboardShare.php

```diff
@@ -145,6 +150,13 @@ private function deleteContactShareAsContact(Dashboard $dashboard, int $contactI
             );
         }
 
+        $accessGroups = $this->readAccessGroupRepository->findByContact($this->contact);
+        $accessGroupIds = array_map(static fn (AccessGroup $accessGroup): int => $accessGroup->getId(), $accessGroups);
```

Comment:
Could be done in one line as the `$accessGroups` variable is not used elsewhere (that is not mandatory)

Reply:
Seen with adr-mo